### PR TITLE
perf: avoid multiple initLevels calls in a single tick

### PR DESCRIPTION
**Pull request by iProdigy**
Follow-up from #250

### CODE REVIEW

**Review by pajlada — APPROVED** [review]
Code looks good to me, haven't tested

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - Minor: Include link to killed player profile in PK notifier. (#246)
 - Minor: Add setting to exclude group name from GIM shared storage notifications. (#247)
 - Minor: Include relevant wiki links in rich embed content. (#243, #248)
+- Bugfix: Avoid combat level notifications that don't conform to the configured interval. (#250, #251)
 - Bugfix: Ignore notifications from new beta worlds. (#253)
 - Bugfix: Avoid combat level notifications that don't conform to the configured interval. (#250)
```
Comment (pajlada) [inline CHANGELOG.md:8]: The changelog entry is duplicated

**Review by pajlada — CHANGES_REQUESTED** [review]
Code looks good, just changelog nitpick

**Review by pajlada — APPROVED** [review]

**Review by Felanbird — APPROVED** [review]
I attempted to review this PR but then after I finished realized I was still on the current master from trying to replicate the bug(s) that this PR fixes.

So we're just gonna say this works and hope for the best 🤞

**Pull request merged**